package.json

```diff
@@ -21,7 +22,10 @@
   "dependencies": {
     "babel": "^6.23.0",
     "express": "^4.16.2",
+    "faker": "^4.1.0",
+    "mongoose": "^5.4.20",
     "morgan": "^1.9.0",
+    "node": "^11.12.0",
```

Comment:
Node shouldn't be a dependency. There is actually a place for to specify node versions in the package.json